source/MaterialXGenGlsl/GlslResourceBindingContext.cpp

```diff
@@ -32,6 +32,19 @@ void GlslResourceBindingContext::emitDirectives(GenContext& context, ShaderStage
 {
     ShaderGenerator& generator = context.getShaderGenerator();
 
+    // write shader stage directives for Vulkan compliance
+    if( _separateBindingLocation )
```

Comment:
Minor: It looks like the coding style here doesn't match the codebase, where we usually have a space after the `if`, and no spaces inside the parentheses.  Once Clang 13 is released, we're planning to add a clang-format file for MaterialX, which should allow for automated style alignment when needed.